services/wfm/src/test/java/org/openkilda/wfm/OFEventWfmTest.java

```diff
@@ -93,6 +102,9 @@ public static void setupOnce() throws Exception {
         ////////
     }
 
+    /**
+     * Clear up everything.
```

Comment:
We should add AfterClass and BeforeClass annotations as exceptions for javadoc check.